### DISCUSSION TITLE
https://raw.githubusercontent.com does not have a certificate

### DISCUSSION THREAD

**Discussion by manu-27993** [community/community#43324] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Cannot open any raw content hosted on github as `https://raw.githubusercontent.com` is not providing an SSL certificate for handshake.

`Microsoft Edge`:
<img width="326" alt="Screenshot 2023-01-04 230917" src="https://user-images.githubusercontent.com/2261081/210616410-c7ca9a0a-7365-47dc-9219-d239d6b7c768.png">

`Google Chrome`:
<img width="269" alt="Screenshot 2023-01-04 231126" src="https://user-images.githubusercontent.com/2261081/210616631-3e4263ba-fd60-4011-b0d7-0e5e089ef953.png">

`curl -l -v https://raw.githubusercontent.com`:
*   Trying 202.83.21.14:443...
* Connected to raw.githubusercontent.com (202.83.21.14) port 443 (#0)
* schannel: disabled automatic use of client certificate
* ALPN: offers http/1.1
* schannel: next InitializeSecurityContext failed: SEC_E_ILLEGAL_MESSAGE (0x80090326) - This error usually occurs when a fatal SSL/TLS alert is received (e.g. handshake failed). More detail may be available in the Windows System event log.
* Closing connection 0
curl: (35) schannel: next InitializeSecurityContext failed: SEC_E_ILLEGAL_MESSAGE (0x80090326) - This error usually occurs when a fatal SSL/TLS alert is received (e.g. handshake failed). More detail may be available in the Windows System event log.

**Comment by hstejas** [comment, marked as answer]
This seems to have been blocked by Govt of India according to act 

https://news.abplive.com/technology/github-content-domain-blocked-india-act-fibernet-broadband-bengaluru-1573258

```
$ curl http://raw.githubusercontent.com
<html>
<head>
<title>Court Orders</title>
</head>
<body>

<h4>This URL has been blocked under the instructions in compliance with the orders of a Hon'ble Court.</h4>

</body>
</html>
```
